cards.js

```diff
@@ -116,6 +116,15 @@ class CardSet {
     }
   }
 
+  includes(card) {
+    let result = this.cards.filter(x => {return (x.suit == card.suit && x.rank == card.rank)})[0] ? true : false; //filter out cards that match and select the 1st element, if it exists, result is true, if it doesn't, result is false
```

Comment:
the `{return ..}` is redundant, can be just `x => x.suit === card.suit && ..`

Reply:
Did not know that, done.